docs/sql-reference/sql-statements/data-types/BINARY.md

```diff
@@ -0,0 +1,146 @@
+# BINARY/VARBINARY
+
+
+## Description
+
+BINARY(M)
+
+VARBINARY(M)
+
+1. Since StarRocks version 3.0, StarRocks supports `BINARY`/`VARBINARY`, the maximum supported length is the same as `VARCHAR` type, and the value range of `M` is 1~1048576.
+2. `BINARY` is just an alias of `VARBINARY`, the usage is exactly the same as `VARBINARY`;
+3. `BINARY(M)`/`VARBINARY(M)` will not pad the unaligned length;
+
+## 示例
+
+### Create a column of type BINARY
+
+When creating a table, use the keyword `BINARY` to specify column `j` as BINARY type.
+
+```SQL
+CREATE TABLE `test_binary` (
+    `id` INT(11) NOT NULL COMMENT "",
+    `j`  VARBINARY NULL COMMENT ""
+) ENGINE=OLAP
+DUPLICATE KEY(`id`)
+COMMENT "OLAP"
+DISTRIBUTED BY HASH(`id`) BUCKETS 1
+PROPERTIES (
+    "replication_num" = "1",
+    "in_memory" = "false",
+    "storage_format" = "DEFAULT"
+);
+
+mysql> desc test_binary;
++-------+-----------+------+-------+---------+-------+
+| Field | Type      | Null | Key   | Default | Extra |
++-------+-----------+------+-------+---------+-------+
+| id    | int       | NO   | true  | NULL    |       |
+| j     | varbinary | YES  | false | NULL    |       |
++-------+-----------+------+-------+---------+-------+
+2 rows in set (0.01 sec)
+
+```
+
+### Import data and store it as BINARY type
+
+StarRocks supports the following three ways to import data and store it as BINARY type.
+
+- Method 1: Use `INSERT INTO` to write data to a constant column of BINARY type (such as column `j`), where the constant column is prefixed with `x'``.
```

Comment:
how to load binary data type from parquet ?

Reply:
This PR is only for StorRocks's inner table usage. Can I add the external table(parquet)'s binary type in the later pr? 

I know parquet's binary type should be supported yet.

Reply:
Not the external table, but the broker load from parquet file

Reply:
Thanks for your replies. Update docs. PTAL